Gemfile

```diff
@@ -0,0 +1,10 @@
+source "http://rubygems.org"
+
+group :development do
+  gem 'therubyracer'
+  gem 'jekyll'
+  gem 'kramdown'
+  gem 'pygments.rb'
+  gem 'rake'
+  gem 'html-proofer'
+end
```

Comment:
Jekyll? https://github.com/Katello/katello.org/blob/deploy/deploy.rb#L127

I can probably get by without rake and html-proofer.